src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -72,6 +81,14 @@ public void setup() {
     // ServiceManager?
   }
 
+  private LoggingServiceBuilder getLoggingServiceBuilder() {
+    return LoggingService.builder()
+        .successfulResponseLogLevel(LogLevel.DEBUG)
```

Comment:
Not logging any successful response , say prom scraping `/metrics` every 30 seconds at INFO

Reply:
Please make this a code comment, so we will know the reason we set it to DEBUG level in the code in the future.